api/src/models/brand.ts

```diff
@@ -10,6 +19,15 @@ export default class Brand extends BaseEntity {
   @Column()
   name!: string
 
+  @CreateDateColumn({ name: 'created_at' })
```

Comment:
Do we need to rename it to something snakecase ? Can't we let it camelCase like we did with the other fields, for exemple productId ?

Reply:
Yes i made the change, it's now in camelCase as typescript property since we did camelCase for FK fields too.